unit03/strings.rst

```diff
@@ -0,0 +1,42 @@
+Unit 3 - Strings Learning Objectives
+====================================
+
+Goals
+-----
+
+- Understand that strings are ordered *collections*.
+- Work with strings using bracket notation, iteration, and common string
+  methods.
+- Compare strings to each other.
+
+Objectives
+----------
+
+Students will:
+
+- Understand and correctly interpret **zero-based indexing**.
+- Understand that strings are indexed collections.
+- Use bracket notation and index values to access a subset of a string.
+- Use iteration to loop through the characters within a string.
+- Use the **accumulator pattern** to build a new string.
+- Describe what it means for strings to be **immutable**.
+- Recognize that reassigning a new string to a variable does not violate the
+  idea of string immutability.
+- Find the length of a string using the ``len()`` function.
+- Create strings with the newline and tab characters: ``\n``, ``\t``.
+- Include quotes within strings.
+- Correctly use the ``+`` and ``*`` operators to concatenate strings.
+- Use template literals to create strings that rely on variable values.
+- Recognize that string methods return new values without changing the
+  original string.
+- Use the ``format()`` method to insert values into a string.
+- Use other common string methods: ``count()``, ``find()``, ``index()``,
+  ``lower()``, ``replace()``, ``strip()``, ``upper()``.
+- Use ``split()`` and ``list()`` to convert a string into a list of string
+  elements.
+- Use boolean methods like ``isalpha()`` and ``islower()`` to check the state
```

Comment:
isalpha() doesnt look like its in the book so I'd remove this

Reply:
Removed.